### DISCUSSION TITLE
There are few documents and examples

### DISCUSSION THREAD

**Discussion by alexzchen** [dlemstra/Magick.NET#721] [Help]
Magick.Net Is a very powerful tool, but there are few documents and examples. It has too many methods and properties, but I don't know how to use it. Can the author improve the document for Imagek.Net and provide some advanced examples, such as channels, text effects, hue processing, transformation and so on. 
In addition, I also want to ask, can I do all the things in document (https://www.imagemagick.org/Usage/)  with Magick.Net，or only support a part of them?

**Comment by dlemstra** [comment]
You can do almost all of the examples of the Usage page with Magick.NET there are only a few exceptions. And I am the only person that works on this project and I don't have enough time to create a new set of documentation. There are a few examples but I could really use some help to write some more examples.
